lib/browser/api/web-contents.js

```diff
@@ -286,6 +286,17 @@ WebContents.prototype.loadFile = function (filePath, options = {}) {
   }))
 }
 
+WebContents.prototype.setWindowOpenOverride = function (handler) {
+  this.windowOpenOverrideHandler = handler
```

Comment:
since this variable isn't supposed to be accessed directly, maybe
```suggestion
  this._windowOpenOverrideHandler = handler
```